enterprise/server/ociregistry/ociregistry.go

```diff
@@ -217,6 +229,33 @@ func (r *registry) handleBlobRequest(ctx context.Context, w http.ResponseWriter,
 		return
 	}
 
+	var rc io.ReadCloser
+
+	casDigest := &repb.Digest{
+		Hash:      blobSHA256Digest, // your SHA256 string
+		SizeBytes: blobSize,         // the size of your blob
+	}
+	bsClient := r.env.GetByteStreamClient()
+	resourceName := digest.NewResourceName(
+		casDigest,
+		"",
+		rspb.CacheType_CAS,
+		repb.DigestFunction_SHA256,
+	)
+	buf := bytes.NewBuffer(nil)
```

Comment:
I don't think it's good idea to load the entire layer into memory. Can you stream the data instead?

Reply:
Yes. I now pass a `Writer` into `GetBlob`. Take a look when you get a chance? 🙏